src/winetricks

```diff
@@ -16127,6 +16127,55 @@ load_wmp10()
     w_set_winver 'default'
 }
 
+w_metadata wmp11 dlls \
+    title="Windows Media Player 11" \
+    publisher="Microsoft" \
+    year="2006" \
+    media="download" \
+    file1="wmp11-windowsxp-x64-enu.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Windows Media Player/wmplayer.exe"
+
+load_wmp11()
+{
+    #w_package_unsupported_win64
```

Comment:
You probably want w_package_unsupported_win32 instead (or does it support 32-bit prefixes).

Reply:
Done.